doc/source/contribution/documentation/introduction.rst

```diff
@@ -15,25 +15,42 @@ Groonga uses Sphinx_ as documentation tool.
 
 Here are command lines to install Sphinx.
 
-Debian GNU/Linux, Ubuntu::
+Debian GNU/Linux, Ubuntu:
 
-  % sudo apt-get install -V -y python3-pip gettext
-  % sudo pip install -r doc/requirements.txt
-  % (cd doc && bundle install)
+.. code-block:: console
 
-CentOS, Fedora::
+   % sudo apt-get install -V -y python3-pip gettext
+   % sudo pip install -r doc/requirements.txt
+   % (cd doc && bundle install)
 
-  % sudo yum install -y python-pip gettext
-  % sudo pip install -r doc/requirements.txt
-  % (cd doc && bundle install)
+CentOS, Fedora:
 
-OS X::
+.. code-block:: console
 
-  % brew install python
-  % brew install gettext
-  % export PATH=`brew --prefix gettext`/bin:$PATH
-  % pip install -r doc/requirements.txt
-  % (cd doc && bundle install)
+   % sudo yum install -y python-pip gettext
+   % sudo pip install -r doc/requirements.txt
+   % (cd doc && bundle install)
+
+OS X:
+
+.. code-block:: console
+
+   % brew install python
+   % brew install gettext
+   % export PATH=`brew --prefix gettext`/bin:$PATH
+   % pip install -r doc/requirements.txt
+   % (cd doc && bundle install)
+
+How to Install required packages for generating documents
+---------------------------------------------------------
+
+You can install required packages for generating Groonga documentation by the following command.
+
+Debian GNU/Linux, Ubuntu:
+
+.. code-block:: console
+
+   % ./setup.sh
```

Comment:
Probably, user can not found `setup.sh` in this description alone.
For example, I think that we should add the following procedure.

```
% git clone ...
% cd groonga/
% ./setup.sh
```

Reply:
At https://github.com/groonga/groonga/pull/1712, I added the following commands at the beginning of this page so I think users have enough information about it. What do you think of it?
```console
% git clone --recursive https://github.com/${YOUR_GITHUB_ACCOUNT}/groonga.git
% cd groonga
```
ref: https://github.com/otegami/groonga/blob/improve-how-to-contribute-in-docs/doc/source/contribution/documentation/introduction.rst#how-to-clone-groonga-repository

Reply:
Or how about we will write it like the following?
- I think the `git clone..` command is a little bit too much because the user wants to know the path at this point.
```console
% cd groonga
% ./setup.sh
``

Reply:
Ah.I see.
It's fine like this.

Reply:
fix: [c91ac05](https://github.com/groonga/groonga/pull/1715/commits/c91ac0558d819d8e5ae0864876071214d8d71461) I add the `cd groonga` command because it's hard for users to find the `setup.sh` file.